src/pages/RegistrationPage/RegistrationPage.js

```diff
@@ -107,7 +113,7 @@ function RegistrationPage() {
                 label="Enter your last name"
```

Comment:
I suggest for all field just leave text 'Last name' instead of 'Enter your last name', as I did in design

Reply:
Noted